api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java

```diff
@@ -21,7 +21,8 @@
 
 /**
  * Injects and extracts a value as text into carriers that travel in-band across process boundaries.
- * Values are often encoded as RPC/HTTP request headers.
+ * Encoding is expected to conform to the HTTP/1.1 Header Field semantics. Values are often encoded
+ * as RPC/HTTP request headers.
  *
  * <p>When using http, the carrier of propagated data on both the client (injector) and server
```

Comment:
perhaps beggiing of this sentence is not needed here

Reply:
"Encoding is expected to conform to the HTTP/1.1 Header Field semantics." ? 

We need to make sure that users know this, and why we don't have separated text-map/http-headers formats.

Reply:
It was a comment about this:

> When using http,

Reply:
Aha, gotcha, my bad. Makes sense ;)